sway/commands.c

```diff
@@ -166,7 +166,7 @@ static bool cmd_exit(struct sway_config *config, int argc, char **argv) {
 	}
 	// Close all views
 	container_map(&root_container, kill_views, NULL);
-	exit(0);
+	wlc_terminate();
```

Comment:
Well, it's an abort after all.
But that reminds me to mark IPC client sockets as CLOEXEC (or does wlc do that?)